neuralop/models/gino.py

```diff
@@ -13,7 +13,9 @@
 from ..layers.gno_block import GNOBlock
 
 class GINO(nn.Module):
-    """GINO: Geometry-informed Neural Operator
+    """GINO: Geometry-informed Neural Operator. Learns a mapping between
+        functions defined over arbitrary coordinate meshes with spectral convolution
```

Comment:
they are note defined over arbitrary coordinate, they are presented over arbitrary coordinate meshes. 

Also, with may refer to the functions. Put a period, and say the model carry global integration through spectral convolution.